packages/portal-proto/src/features/GenomicTables/SomaticMutationsTable/SMTableContainer.tsx

```diff
@@ -164,8 +164,42 @@ export const SMTableContainer: React.FC<SMTableContainerProps> = ({
         "gene.ssm.ssm_id",
       );
     }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [topSSM]);
+  }, [
+    data,
```

Comment:
eslint warning:` useDeepCompareCallback has unnecessary dependencies: 'data' and 'searchTerm'. Either exclude them or remove the dependency array`

So remove data and searchTerm

Reply:
Pushed the change